### PR TITLE
Rucio download is not able to download files if the scope or name contains "/"

### PR DESCRIPTION
**Pull request by cserf**
Rucio download is not able to download files if the scope or name contains "/"

### CODE REVIEW

File: lib/rucio/client/downloadclient.py [hunk 1]
```diff
@@ -233,7 +236,10 @@ def download_pfns(self, items, num_threads=2, trace_custom_fields={}, traces_cop
             item['scope'] = did_scope
             item['name'] = did_name
             item['sources'] = [{'pfn': pfn, 'rse': rse}]
-            dest_file_path = os.path.join(dest_dir_path, did_name)
+            did_path_name = did_name
+            if self.extract_scope_convention == 'belleii' and did_name.startswith('/'):
```
Comment (mlassnig) [inline lib/rucio/client/downloadclient.py]: extract_scope_convention can be None (as per config_get above), then this will equality check will fail

Comment (mlassnig) [reply lib/rucio/client/downloadclient.py]: @bari12 maybe we should think of a way of abstracting this? having vo specific checks inside the generic download() is maybe not so good

**Review by mlassnig — COMMENTED** [review]
two comments inside

**Comment by bari12** [comment]
I'll merge it like this, but we need to think about reasonable ways to get this VO-specific code out of the code-base. 
For the server/daemon side, we have the policy modules, we should do something similar on client side as well.

**Review by bari12 — APPROVED** [review]

**Pull request merged**